src/components/TodoFooter.tsx

```diff
@@ -0,0 +1,69 @@
+/* eslint-disable import/no-cycle */
+import { useContext } from 'react';
+import classNames from 'classnames';
+import { Filter } from '../types/enum';
+import { TodoContext } from '../TodoContext';
+import { TodoContextProps } from '../types/interfaces';
+
+export const TodoFooter = () => {
+  const { state, setState } = useContext(TodoContext) as TodoContextProps;
+
+  const itemsLeft = state.todos.filter(todo => !todo.completed).length;
```

Comment:
what items?
maybe its better to update to `uncompletedTodosLeft`, `incompleteTodosCount`